### PR TITLE
[RANS] RANS Turbulence models integration - Step 3.4.3 - Adding VMS Monolithic Formulation

### PR DESCRIPTION
**Pull request by sunethwarna**
**Description**
This add VMS Monolithic Formulation following #7304. Description is given in #7304

**Changelog**
- Added VMS Monolihtic formulation
- Added tests

### CODE REVIEW

**Review by philbucher — COMMENTED** [review]
:+1:

**Review by rubenzorrilla — APPROVED** [review]

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]
Wrong PR sorry

**Comment by sunethwarna** [comment]
This is ready for review @rubenzorrilla :)

File: applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+            conv_criteria,
+            builder_and_solver,
+            settings["maximum_iterations"].GetInt(),
+            settings["compute_reactions"].GetBool(),
+            settings["reform_dofs_at_each_step"].GetBool(),
+            settings["move_mesh_flag"].GetBool())
+
+        builder_and_solver.SetEchoLevel(self.echo_level - 3)
+        self.solver.SetEchoLevel(self.echo_level - 2)
+        conv_criteria.SetEchoLevel(self.echo_level - 1)
```
Comment (rubenzorrilla) [inline applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py]: This changes the expected output for a given echo level. I'd keep the current ones. If these are not suitable, we can open a discussion to reduce the current verbosity.

File: applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    def Clear(self):
+        self.solver.Clear()
+        super().Clear()
+
+    def SetTimeSchemeSettings(self, settings):
+        if (settings.Has("scheme_type")):
+            scheme_type = settings["scheme_type"].GetString()
+            if (scheme_type == "steady"):
+                self.is_steady_simulation = True
+            elif (scheme_type == "transient"):
```
Comment (rubenzorrilla) [inline applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py]:
```suggestion
            elif (scheme_type == "bossak"):
```

File: applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+                self.is_steady_simulation = False
+                default_settings = Kratos.Parameters('''{
+                    "scheme_type": "transient",
+                    "alpha_bossak": -0.3
+                }''')
+                settings.ValidateAndAssignDefaults(default_settings)
+                self.GetBaseModelPart().ProcessInfo.SetValue(Kratos.BOSSAK_ALPHA, settings["alpha_bossak"].GetDouble())
+            else:
+                raise Exception("Only \"steady\" and \"transient\" scheme types supported. [ scheme_type = \"" + scheme_type  + "\" ]")
```
Comment (rubenzorrilla) [inline applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py]:
```suggestion
                raise Exception("Only \"steady\" and \"bossak\" scheme types supported. [ scheme_type = \"" + scheme_type  + "\" ]")
```

**Review by rubenzorrilla — DISMISSED** [review]
Only minor comments.

File: applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py (hunk 1 above)
Comment (sunethwarna) [reply applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py]: done :)

File: applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py (hunk 2 above)
Comment (sunethwarna) [reply applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py]: done :)

File: applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py (hunk 3 above)
Comment (sunethwarna) [reply applications/RANSApplication/python_scripts/formulations/monolithic_vms/monolithic_velocity_pressure_rans_formulation.py]: done :)

**Comment by sunethwarna** [comment]
@rubenzorrilla Could you reapprove? I applied your suggestions :)

**Review by rubenzorrilla — APPROVED** [review]

**Comment by sunethwarna** [comment]
Thanks @rubenzorrilla  :)

**Pull request merged**